doc/changelog.md

```diff
@@ -276,36 +301,79 @@
   Array([0.99500417, 0.98006658, 0.95533649], dtype=float64)
   ```
 
-* Verification is now performed before compilation to confirm that the measurements included in the quantum tape
-  are compatible with the device.
+* Verification is now performed before compilation to confirm that the measurements included in QNodes
+  are compatible with the specified device and settings.
   [(#945)](https://github.com/PennyLaneAI/catalyst/pull/945)
   [(#962)](https://github.com/PennyLaneAI/catalyst/pull/962)
 
-* Update JAX to `v0.4.28`.
-  [(#931)](https://github.com/PennyLaneAI/catalyst/pull/931)
-  [(#995)](https://github.com/PennyLaneAI/catalyst/pull/995)
-
-* Adds `catalyst.from_plxpr.from_plxpr` for converting a PennyLane variant jaxpr into a
-  Catalyst variant jaxpr.
-  [(#837)](https://github.com/PennyLaneAI/catalyst/pull/837)
+  ```pycon
+  >>> dev = qml.device("lightning.qubit", wires=2, shots=None)
+  >>> @qjit
+  ... @qml.qnode(dev)
+  ... def circuit(params):
+  ...     qml.RX(params[0], wires=0)
+  ...     qml.RX(params[1], wires=1)
+  ...     return {
+  ...         "sample": qml.sample(wires=[0, 1]),
+  ...         "expval": qml.expval(qml.PauliZ(0))
+  ...     }
+  >>> circuit([0.1, 0.2])
+  CompileError: Sample-based measurements like sample(wires=[0, 1])
+  cannot work with shots=None. Please specify a finite number of shots.
 
 * On devices that support it, initial state preparation routines `qml.StatePrep` and `qml.BasisState`
-  are no longer decomposed when using Catalyst, improving compilation & runtime performance.
+  are no longer decomposed when using Catalyst, improving compilation and runtime performance.
   [(#955)](https://github.com/PennyLaneAI/catalyst/pull/955)
 
+* Catalyst is now compatible with Enzyme `v0.0.130`
+  [(#898)](https://github.com/PennyLaneAI/catalyst/pull/898)
```

Comment:
@rmoyard are there any user benefits I can mention here? Or would this best fit under 'internal changes'?

Reply:
You can move it in internal changes :+1: